README.md

```diff
@@ -1,3 +1,7 @@
+#---------
+# THIS IS NO LONGER USED - see the lambdas in the api layer https://github.com/guardian/zuora-auto-cancel
+#---------
+
```

Comment:
that's where the /subs endpoint is now.
/auth and the api gateway are defined here: 
https://github.com/guardian/digital-subscription-authorisation